resources/visualization:Replayed-Traffic-Dashboard-Description.json

```diff
@@ -1,6 +1,6 @@
 {
    "migrationVersion": {
-      "visualization": "7.2.0"
+      "visualization": "7.4.2"
```

Comment:
Why did all of these change to 7.4.2? Those changes seem like they should be reverted.